client/scss/settings/_variables.scss

```diff
@@ -31,57 +31,35 @@ $breakpoints: (
 );
 
 // colours
-$color-teal-light: var(--color-primary-light);
-$color-teal-lighter: var(--color-primary-lighter);
-$color-teal: var(--color-primary);
-$color-teal-darker: var(--color-primary-darker);
-$color-teal-dark: var(--color-primary-dark);
-
-$color-blue: #71b2d4;
-$color-red: #cd3238;
-$color-red-dark: #b4191f;
-$color-orange: #e9b04d;
-$color-orange-dark: #bb5b03;
-$color-green: #189370;
-$color-green-dark: #157b57;
-$color-green-darker: #105d42; // White has AAA compatibility when this is used in bgd.
+$color-teal: theme('colors.secondary.DEFAULT');
+$color-teal-darker: theme('colors.secondary.400');
+$color-teal-dark: theme('colors.secondary.600');
+
 $color-salmon: #f37e77;
 $color-salmon-light: #fcf2f2;
-$color-white: #fff;
-$color-black: #000;
+$color-white: theme('colors.white.DEFAULT');
+$color-black: theme('colors.black.DEFAULT');
 
 // darker to lighter
-$color-grey-1: color.adjust($color-white, $lightness: -80%);
-$color-grey-2: color.adjust($color-white, $lightness: -70%);
-$color-grey-3: color.adjust($color-white, $lightness: -15%);
-$color-grey-4: color.adjust($color-white, $lightness: -10%);
-$color-grey-5: color.adjust($color-white, $lightness: -2%);
-
-$color-menu-text: color.adjust($color-white, $lightness: -20%);
+$color-grey-1: theme('colors.grey.600');
+$color-grey-2: theme('colors.grey.400');
+$color-grey-3: theme('colors.grey.200');
+$color-grey-4: theme('colors.grey.100');
+$color-grey-5: theme('colors.grey.50');
```

Comment:
Yes, I’d prefer to _at least_ rename the "Grey" scale to follow the same naming as the design tokens, however this would change even more lines in the stylesheets, so I decided not to touch those variables until we agree on how to name them exactly.